src/NuGet.Clients/NuGetConsole.Host.PowerShell/PowerShellHost.cs

```diff
@@ -484,6 +499,11 @@ private async Task ExecuteInitPs1Async(string installPath, PackageIdentity ident
                             request.BuildInput(),
                             outputResults: true);
 
+                        var telemetryEvent = new TelemetryEvent(eventName: "PowershellScriptExecuted");
+                        telemetryEvent.AddPiiData("id", identity.Id?.ToLowerInvariant() ?? "(empty package id)");
```

Comment:
What's the value of getting the ids?

Reply:
Currently no immediate use. But during last Teams call someone mentioned we're not tracking this events, specially since EF6 one of most used package we need to pay attention to how people use it.
There could be other packages use powershell init.ps1 extensively. I believe we need this information later when we analyze use cases of PMC with powershell.

Reply:
nit: PowerShell

Reply:
Ok. Fixed it.

Reply:
Removed since there is no planned use for now. Also it doesn't capture all cases.